### ISSUE TITLE
Multiple errors converting .net core to csproj

### ISSUE THREAD

**Issue by 333fred** [dotnet/sdk#447]
Moving https://github.com/dotnet/cli/issues/4871 on behalf of @gilmishal.

> I have opened this [SO](http://stackoverflow.com/questions/40892034/multiple-errors-converting-net-core-to-csproj) question, and decided it might be worth asking here as well.

> I have migrated an asp.net core project from xproj to csproj for using VS2017RC. Now everything went to hell... getting a lot of msbuild errors, Visual Studio 2015 can't load nuget packages. VS2017 manages to build the project - but Intellisense is going crazy..

Copy of the SO post:
> After installing VS2017 RC I realized I needed to upgrade my asp.net core project to work with csproj instead of xproj and project.json.

> The upgrade went well, but now I am getting a lot of warnings, and an error.

> first, there are warnings regarding the .csproj file. I am getting the following warning - even in a new core class library.

> ```
> The element 'PropertyGroup' in namespace 'http://schemas.microsoft.com/developer/msbuild/2003' has > invalid child element 'TargetFramework' in namespace > > 'http://schemas.microsoft.com/developer/msbuild/2003'.
> ```
> I am also getting
> ```
> The OutputPath is not set for project
> ```
> Adding an OutputPath in the .csproj file fixes that error, but the OutputPath property is not set in the new class library project I created and it still compiles.

> Even after setting the OutputPath, I am getting other errors and warnings, but they don't affect the build (however - and I can't seem to reproduce them in a new project) and even though intellisense is going crazy.

> other than that - I am getting over a thousand errors in VS2015 (it can't really load the nuget packages for some reason).

> It might be worth mentioning that I am trying to target both .net core and .net461

**Comment by 333fred** [comment]
An important note for migration is that it is one way: VS2015 does not have the new project system, and cannot open migrated projects.

Can you point us at the unmigrated solution?

**Comment by gilmishal** [comment]
I can't really do that since the code isn't public.

I can show you the migrated csproj and the unmigrated package.json.

I will do that tomorrow though.

I actually have an entire solution that I tried to migrate, but I will try to explain to you better what I managed to understand of what happend, after an entire day of trying to fix it.

I will begin with the fact that I tried to update to asp.net core 1.1.0 I downloaded the .net core 1.1.0 sdk - it wouldn't build with xproj, so I had to migrate using .net migrate. I did try to use the global.json specifying the older sdk, but that didn't work.
At the end I found out I didn't exactly have to - building and restoring from the cli worked, only VS2015 wouldn't restore packages correctly.

Anyway, since I didn't know the problem was with visual studio and it said the xproj is not supported I decided to migrate - via the cli.

After changing the resulting csproj I managed to get it to work on VS2017 - however, I couldn't reference other projects in my solution (I can with a new project) and I can't target .net4.6.1 which is necessary for me. 

I actually think that the fact that I target both .net framework and .net core resulted in a faulty migration.
